doc/en/installation/from_packages.rst

```diff
@@ -21,153 +21,144 @@ SELinux should be disabled; for this, you have to modify the file */etc/selinux/
-Install a central server
-************************
+To install Centreon software from the repository, you should first install centreon-release package
+which will provide the repository file.
 
-The chapter describes the installation of a Centreon central server.
+Perform the following command with an user granted with sufficient rights.
 
-Perform the command:
+Centreon repository installation.
 
 ::
 
-  $ yum install centreon-base-config-centreon-engine centreon
-
-:ref:`After this step you should connect to Centreon to finalise the installation process <installation_web_ces>`.
+   $ wget http://yum.centreon.com/standard/18.9/el7/stable/noarch/RPMS/centreon-release-18.9.el7.centos.noarch.rpm
+   $ yum install --nogpgcheck centreon-release-18.9.el7.centos.noarch.rpm
 
-Installing a poller
---------------------
 
-This chapter describes the installation of a collector.
+The repository is now installed.
 
-Perform the command:
 
- ::
+************************************
+Installing a Centreon central server
+************************************
 
- $ yum install centreon-poller-centreon-engine
+This chapter describes the installation of a Centreon central server.
 
-The communication between a central server and a poller server is by SSH.
+Perform the command:
 
-You should exchange the SSH keys between the servers.
+::
 
-If you don’t have any private SSH keys on the central server for the Centreon user:
+  $ yum install centreon-base-config-centreon-engine centreon
 
- ::
+Installing MySQL on the Centreon central server
+-----------------------------------
 
- $ su - centreon
- $ ssh-keygen -t rsa
+This chapter describes the installation of MySQL on a server including Centreon.
 
-Copy this key on the collector:
+Perform the command:
 
- ::
+::
 
- $ ssh-copy-id centreon@your_poller_ip
+   $ yum install MariaDB-server
+   $ systemctl restart mysql
 
+Database management system
+--------------------------
 
-Installing MySQL on the same server
------------------------------------
+The MySQL database server should be available to complete installation (locally or not). MariaDB is recommended.
 
-This chapter describes the installation of MySQL on a server including Centreon.
+It is necessary to modify **LimitNOFILE** limitation.
+Setting this option into /etc/my.cnf will NOT work.
 
-Perform the command:
+Perform instead:
 
 ::
 
-   $ yum install MariaDB-server
-   $ service mysql restart
-
+   # mkdir -p  /etc/systemd/system/mariadb.service.d/
+   # echo -ne "[Service]\nLimitNOFILE=32000\n" | tee /etc/systemd/system/mariadb.service.d/limits.conf
+   # systemctl daemon-reload
+   # service restart mysql
 
 PHP timezone
 ------------
 
-PHP timezone should be set; go to /etc/php.d directory and create a file named php-timezone.ini which contains the following line :
+PHP timezone needs to be set; to do so go to /etc/opt/rh/rh-php71/php.d directory and create a file named php-timezone.ini which contains the following line :
```

Comment:
which contains the following line : -> which contains the timezone of the centreon's server. For example :

Reply:
done